parametric/test_span_sampling.py

```diff
@@ -526,3 +526,148 @@ def test_child_span_selected_by_sss015(test_agent, test_library):
     assert child_span["metrics"].get(SINGLE_SPAN_SAMPLING_RATE) == 1.0
     assert child_span["metrics"].get(SINGLE_SPAN_SAMPLING_MECHANISM) == SINGLE_SPAN_SAMPLING_MECHANISM_VALUE
     assert child_span["metrics"].get(SINGLE_SPAN_SAMPLING_MAX_PER_SEC) == 50
+
+
+@pytest.mark.skip_library("golang", "The Go tracer sends the full trace to the agent anyways.")
```

Comment:
Does Go Tracer or .NET Tracer support dropping on the tracer side? If so, how to enabled it, so these new tests pass for Go/.NET.